test/int/deployment_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"strconv"
 	"time"
 
+	retry2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/retry"
```

Comment:
nitpick
may I suggest
```suggestion
	akoretry "github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/retry"
```

Reply:
good catch, I chose `testretry` though as this is explicitly a test helper.

Reply:
eh, no, `akoretry` is better 👍